webpack.config.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
   entry: {
     index: ['./web_src/js/index'],
     swagger: ['./web_src/js/swagger'],
+    jquery: ['./web_src/js/jQuery'],
```

Comment:
filename should probably be lowercase:
```suggestion
    jquery: ['./web_src/js/jquery'],
```

Reply:
Actually, camel case is correct here and matches the style in the ` web_src/js` directory.

Reply:
imho they all should be renamed :laughing: 
but as for jquery I was more thinking of consistency between src and target name

Reply:
agree, will change it to lowercase.